src/dataFetcher.js

```diff
@@ -0,0 +1,25 @@
+import fs from "fs";
+import readLine from "readline";
+
+const parse = line => {
+    const parameters = line.split("\t");
```

Comment:
Why not use a CSV parser?
Parsing CSV files is seemingly simple but there are many edge-cases that available CSV parser libs handle.